devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -457,7 +457,10 @@ private void addLocalProject(Project project, DevModeContext context, Set<AppArt
                 sourcePaths,
                 classesDir,
                 resourcesSrcDir.getAbsolutePath(),
-                resourcesOutputPath);
+                resourcesOutputPath,
+                Paths.get("src", "main").toAbsolutePath().toString(),
```

Comment:
I guess you should probably be using/checking sourcePaths. Most of the time it will be a singleton set.

Reply:
will do